.github/workflows/slow_tests_gaudi2.yml

```diff
@@ -60,7 +60,7 @@ jobs:
             --net=host \
             --ipc=host \
             vault.habana.ai/gaudi-docker/1.19.0/ubuntu22.04/habanalabs/pytorch-installer-2.5.1:latest \
-            /bin/bash tests/ci/slow_tests_diffusers.sh
+            /bin/bash tests/ci/slow_tests_diffusers.sh ${{ secrets.TEXT_GENERATION_CI_HUB_TOKEN }}
```

Comment:
I wonder if the runner already has a token configured in it's default environment.  @regisss?

How about set it on the container environment instead of passing it to the script (i.e. `-e HF_TOKEN=${{secrets.TEXT_GENERATION_CI_HUB_TOKEN}}`?

Reply:
@uartie Thanks Artie.  There are few other places which use this approach to pass token, e.g. see [here](https://github.com/huggingface/optimum-habana/blob/main/tests/ci/slow_tests_deepspeed.sh).  We should agree on the best method for all, and update all cases.  @regisss WDYT?  (also someone with admin privileges should add 2 gated models to token from secrets)

Reply:
This token is a secret defined at the level of the optimum-habana repository. So all workflows that are allowed to read tokens will have access to it. I'm the admin of the optimum-habana repo so I have full control over it. It's absolutely fine to use here the same token as for the text-generation tests IMO.
@dsocek What are the gated models you would like to test?